### ISSUE TITLE
Remove duplicate dependency lists in conda environment YAML files

### ISSUE THREAD

**Issue by mfisher87** [nsidc/earthaccess#618]
We could:

* Remove them. Personally, I don't think this pure python project has a reason to include them. We can advise conda users to use conda to create the environment, but use pip to install into it.
* Replace them with environment specification files that don't duplicate the dependency lists:
    ```yaml
    name: earthaccess
    channels:
      - conda-forge
      - nodefaults
    dependencies:
      - python>=3.9
      - pip
      - pip:
          - --editable .[dev]
    ```

**Comment by itcarroll** [comment]
I knew I could find another one fixed by #733. @mfisher87 was on a tear!

**Comment by mfisher87** [comment]
Love it! Thanks for digging for these Ian!!
